tools/makerootfs.sh

```diff
@@ -16,4 +16,5 @@ if [ ! -f "$YMLFILE" ] || [ $# -lt 2 ]; then
     exit 1
 fi
 
-linuxkit build -o - "$YMLFILE" | docker run -v /dev:/dev --privileged -i ${MKROOTFS_TAG} > "$IMAGE"
+: > "$IMAGE"
+linuxkit build -o - "$YMLFILE" | docker run -v /dev:/dev --privileged -v "$IMAGE:/rootfs.img" -i ${MKROOTFS_TAG}
```

Comment:
Is this really so much faster? I believe you, of course, but why would it be?

Reply:
Looks like it is ~2x faster. Especially with ext4. Btw, as you saw -- I actually preserved the functionality -- so if you don't -v "$IMAGE:/rootfs.img" and the like you get the older behavior (and then you can benchmark it ;-)).